cmd/workspace/secrets/overrides.go

```diff
@@ -40,15 +45,14 @@ var putSecretCmd = &cobra.Command{
   and cannot exceed 128 characters. The maximum allowed secret value size is 128
   KB. The maximum number of secrets in a given scope is 1000.
 
-  The input fields "string_value" or "bytes_value" specify the type of the
-  secret, which will determine the value returned when the secret value is
-  requested. Exactly one must be specified.
+  The arguments "string-value" or "bytes-value" specify the type of the secret,
+  which will determine the value returned when the secret value is requested.
```

Comment:
Maybe leave this in?
```suggestion
  which will determine the value returned when the secret value is requested. Exactly one must be specified.
```

Reply:
It's not exactly, because if neither is specified, it will prompt for a value or read one from stdin.